cmd/raidz_test/raidz_test.h

```diff
@@ -34,6 +34,8 @@ static const char *raidz_impl_names[] = {
 	"sse2",
 	"ssse3",
 	"avx2",
+	"neon",
+	"neon2",
```

Comment:
I don't have any hardware to test this patch, but it looks good. Please make sure you run `cmd/raidz_test/raidz_test -S` to unit-test this on supporting hardware. You can add `-v` for more verbose output. This is essentially what `raidz_002_pos` test will run for 5min, but we don't have any arm64 testers either. 
also: see the other comment about naming, to avoid potential confusion with arm32 neon.